core/bootstrap/src/main/java/org/eclipse/dataspaceconnector/system/runtime/BaseRuntime.java

```diff
@@ -48,7 +48,7 @@ protected Monitor getMonitor() {
     /**
      * Main entry point to runtime initialization. Calls all methods.
      */
-    protected void boot() {
+    protected ServiceExtensionContext boot() {
```

Comment:
Just curious, is this a convenience or was it required to return the context?

Reply:
pure convenience. I could also intercept one of the initialization hooks to get the context, which I did in my latest commit.